### ISSUE TITLE
Add assigned staff filter in admin view

### ISSUE THREAD

**Issue by TheZoker** [ThemeAvenue/Awesome-Support#108]

**Comment by julien731** [comment]
Could you detail a little more? For which purpose?

**Comment by TheZoker** [comment]
Just add a dropdown menu next to "All dates", "Any Status", "Any Reply Status" which says, "Any assigned staff". Then you can filter the tickets to show e.g. only all tickets assigned to "John Doe".

**Comment by julien731** [comment]
I don't think we will implement that. What I think we will do is create a sort of user profile, both for users and agents, with a couple of stats of the person and a list of his tickets / assigned tickets.

**Comment by TheZoker** [comment]
But its very useful for an author to see only tickets, that are assigned to him and not all tickets, so this filter would really makes sense...

**Comment by julien731** [comment]
Doesn't the "Agents See All" option suffice? 

![image](https://cloud.githubusercontent.com/assets/1441024/6938579/acf2c1ba-d88e-11e4-9880-83b3636a6970.png)

**Comment by TheZoker** [comment]
Yea but I dont want to always switch the settings, to only see my tickets or all (sometimes all is also helpfully). In my opinion a filter would just fit better

**Comment by SiamKreative** [comment]
I agree it would be great to have the ability to filter stuff quickly. From my understanding, it's pretty complex to customize this "list layout" in WordPress... If you're a WordPress dev yourself, feel free to submit a PR :)

**Comment by tednh** [comment]
This feature has been implemented on recently versions of Awesome Support Plugin.
Regards
Ted
